assets/js/lib/markdown.js

```diff
@@ -255,7 +255,18 @@ function rehypeExternalLinks(options) {
       if (node.properties && node.properties.href) {
         const url = node.properties.href;
 
-        if (isInternalUrl(url)) {
+        if (isPageAnchor(url)) {
+          const search = url.replace("#go-to-definition", "");
```

Comment:
To clarify, I wouldn't have this special-casing in the generic markdown renderer. What I was thinking is this:

```js
// session.js

handleDocumentClick(event) {
  if (event.target.matches("a") && event.target.hash.startsWith("#go-to-definition")) {
    // focus the cell, set insert mode and broadcast event to move cursor
  }

  // ...
}
```

Does that make sense?

Reply:
Much more than I was doing